### DISCUSSION TITLE
JWE support in Rego

### DISCUSSION THREAD

**Discussion by richard-robinson** [open-policy-agent/community#290] [OPA and Rego] [answered]
Hello,

In the latest docs for OPA version 0.45.0 (latest), I see this mentioned: "JWE and JWS JSON Serialization are not supported" (https://www.openpolicyagent.org/docs/latest/faq/#json-web-tokens-jwts)

Does OPA or Rego plan to add JWE support in the new future?

If not, are there any known workarounds. I have some ideas myself, but would be interested to hear what others are doing to get around the lack of JWE support.

-- Richard

**Comment by anderseknert** [comment, marked as answer]
Hi Richard!

There's an old [open issue](https://github.com/open-policy-agent/opa/issues/1131) around this, but it's currently not on the roadmap for the near future. If anyone is motivated to work on it, I'd be happy to help :)
